components/org.wso2.carbon.status.dashboard.core/src/gen/java/org/wso2/carbon/status/dashboard/core/api/WorkersApi.java

```diff
@@ -115,7 +114,7 @@ public Response addWorker(@Context Request request,
             @ApiParam(value = "Worker object that needs to be added." ,required=true) Worker worker
     )
             throws NotFoundException {
-        return delegate.addWorker(worker,getUserName(request));
+        return workersApi.addWorker(worker,getUserName(request));
```

Comment:
Add space after comma in method parameters

Reply:
In all the places in this PR

Reply:
Fixed in 68383c43d1d5379a2979203243b041f32f980adf